apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx

```diff
@@ -74,6 +74,7 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
     const isFirstLevelOrg: boolean = useSelector(
         (state: AppState) => state?.organization?.isFirstLevelOrganization
     );
+    const isSAASDeployment: boolean = useSelector((state: AppState) => state?.config?.ui?.isSAASDeployment);
```

Comment:
Let's use the useUIConfig hook.

```suggestion
    const { isSAASDeployment } = useUIConfig();
```

Reply:
Since this hook is not updated to expose `isSAASDeployment`, let's use the config from store.